### PR TITLE
Migrate tests from app/command_test.go to use testify

### PR DESCRIPTION
**Pull request by chahat-arora**
#### Summary
Migrate tests from "app/command_test.go" to use testify
#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/12103
https://mattermost.atlassian.net/browse/MM-18378

### CODE REVIEW

File: cmd/mattermost/commands/command_test.go [hunk 1]
```diff
@@ -117,14 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
 
 			if testCase.ExpectedErr == "" {
-				if len(cmds) == 0 || cmds[0].Trigger != "testcmd" {
-					t.Fatal("Failed to create command")
-				}
+				assert.NotZero(t,len(cmds),"Failed to create command")
+				assert.NotEqual(t,cmds[0].Trigger,"testcmd","Failed to create command")
```
Comment (mgdelacroix) [inline cmd/mattermost/commands/command_test.go]: This should be an `assert.Equal`, as the condition to the `t.Fatal` was that `cmds[0].Trigger != "testcmd"` originally.

**Comment by hanzei** [comment]
Hey @chahat-arora, Do you have any questions about the feedback?

File: cmd/mattermost/commands/command_test.go (hunk 1 above)
Comment (chahat-arora) [reply cmd/mattermost/commands/command_test.go]: fixed

**Comment by chahat-arora** [comment]
sorry for the delay! Now fixed.

**Pull request merged**